apps/ui-kit/lib/components/super-formatter/props.ts

```diff
@@ -0,0 +1,32 @@
+import { PropType } from "vue"
+import { FormatOptions } from "sql-formatter"
+
+export default {
+  value: String,
+  canAddPresets: Boolean,
+  clipboard: Object,
```

Comment:
I would use the [Clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard) interface for typing the `clipboard`. We also use this type in the [TextEditor](https://github.com/beekeeper-studio/beekeeper-studio/blob/ef6a2bff9286435a403617fe997c8b73fd7b586b/apps/ui-kit/lib/components/text-editor/props.ts#L103) component, so it would be more consistent if we use the same type. :)